src/components/expanded-state/swap-details/SwapDetailsExchangeRow.js

```diff
@@ -121,44 +123,88 @@ const ExchangeIconStack = magicMemo(
   ['protocols']
 );
 
-export default function SwapDetailsExchangeRow({ protocols, testID }) {
+const CrossChainIconStack = magicMemo(
```

Comment:
can we hide it under feature flag?

Reply:
its automatically under feature flag, only shows up when a TX is cross-chain. otherwise the old designs are shown